.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,211 @@
+      options:
+        - <= 1GB
+        - <= 2GB
+        - <= 4GB
+        - \> 4GB
+    validations:
+      required: true
+  - type: dropdown
+    id: cpu-count
+    attributes:
+      label: How many CPU cores / threads are available?
+      description: How many CPU cores or threads can DMS use? If you're using Kubernetes, provide the resource limit here if you set one.
+      options:
+        - <= 1 Core
+        - <= 2 Cores
+        - <= 4 Cores
+        - \> 4 Cores
+    validations:
+      required: true
+  - type: dropdown
+    id: system-type
+    attributes:
+      label: Is DMS running in a virtualized environment?
+      description: Choose the option that your setup is the most similar to. DMS is running on...
+      options:
+        - ... a physical host (with real, physical CPU cores)
+        - ... a Virtual Dedicated Server (VDS) (with real, physical cores)
+        - ... a Virtual Private Server (VPS) (with virtual CPU cores)
+    validations:
+      required: true
+  - type: dropdown
+    id: operating-system
+    attributes:
+      label: What operating system is DMS running on?
+      options:
+        - Linux
+        - macOS
+        - Windows
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: isa
+    attributes:
+      label: What instruction set architecture is DMS running on?
+      options:
+        - x86_64 / AMD64
+        - ARM64 / ARM v8 (and newer)
+        - ARM v7
+        - Unsupported / Other
+    validations:
+      required: true
+  - type: dropdown
+    id: container-orchestrator
+    attributes:
+      label: What container orchestration tool are you using?
+      options:
+        - Docker
+        - Docker Compose
+        - Podman
+        - Kubernetes
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: docker-version
+    attributes:
+      label: Docker version
+      description: Please show us the output of `docker --version` on the host running your DMS container. _Note that maintainers primarily provide support for Docker Engine only (users of `docker` and `docker-compose`)._
+      placeholder: Docker version 20.10.7, build f0df350
+    validations:
+      required: false
+  - type: input
+    id: docker-compose-version
+    attributes:
+      label: Docker Compose version
+      description: Please show us the output of `docker-compose --version` on the host running your DMS container. _Note that maintainers primarily provide support for Docker Engine only (users of `docker` and `docker-compose`)._
+      placeholder: docker-compose version 1.28.6, build 5db8d86f
+    validations:
+      required: false
+  - type: input
+    id: uname-movar
+    attributes:
+      label: The output of `uname -movar`
+      description: Please paste the output of `uname -movar` here. If you're unable to do so (maybe because you're on Windows), tell us the reason for your inability here.
```

Comment:
What happened here? `uname -movr` is pointless if doing `uname -a` as well.

If we just want the kernel version `uname -r` works well enough. If you're fine with the full output shown in your placeholder, keep it at `uname -a`.

Reply:
I messed up. I just want `uname -a` to get the full view. It is also "easier", although I gotta admit this is a somewhat useless point. In the end, this is really here to double-check.